lib/screens/home_screen.dart

```diff
@@ -98,15 +99,16 @@ class _HomeScreenState extends State<HomeScreen> with TickerProviderStateMixin {
                       children: [
                         const Expanded(
                           child: Center(
-                            child: Text(
+                            /*child: Text(
                               "You currently have no tasks to do",
                               style: TextStyle(
                                 fontFamily: 'Inter',
                                 fontWeight: FontWeight.w500,
                                 fontSize: 15.0,
                                 color: Color(0xffA7A7A7),
                               ),
```

Comment:
@GloriaMMuthumbi Do we need this commented code?

Reply:
@skingori - yes, that text will be shown when the fragment is empty(the user hasn't added any to-do tasks). I had commented it out so that i can code the task card component to see how it appears on the fragments.